ImmunosuppressiveDrugs/R4/resources/Questionnaire-R4-ImmunosuppressiveDrugsProgressNote.json

```diff
@@ -57,7 +57,11 @@
           "extension": [
             {
               "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire",
-              "valueCanonical": "questionnaire/encounter-items"
+              "valueCanonical": "questionnaire/encounter"
+            },
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire-expand",
+              "valueBoolean": true
```

Comment:
If this is just for demo purpose, it is fine. We don't really add any additional item here, right? Should we remove the expand true here?

Reply:
Yes. This is for demo.

Reply:
fixed